ext/oci8/oci8_interface.c

```diff
@@ -2189,6 +2197,7 @@ PHP_FUNCTION(oci_result)
    Return a string containing runtime client library version information */
 PHP_FUNCTION(oci_client_version)
 {
+	ZEND_PARSE_PARAMETERS_NONE();
```

Comment:
Nit: This would be nicer to be placed after the declaration.

Reply:
I think it's still not addressed.

Reply:
Hm, that's strange. If you check my last commit you can see the change.

Reply:
Hmm, no idea why GitHub shown me the previous state, but I can definitely see it now. :) Thanks!